CHANGES.md

```diff
@@ -12,6 +12,8 @@
 
 - fixed a crash when PWD=/ on POSIX (#1631)
 
+- fixed "I/O operation on closed file" on non-Windows OS when using --diff (#1664)
```

Comment:
Err, the `I/O operation on closed file` errors were also present on Windows...

Reply:
It was not clear to me that this bug manifested on Windows. Good to know.

I've updated CHANGES.md to reflect this.